### DISCUSSION TITLE
[Feature] iOS app select album cover

### DISCUSSION THREAD

**Discussion by Squ1dw4rd99** [immich-app/immich#14147] [Feature Request]
Feature:
Add the feature to allow users to set a specific photo as the album cover for their albums within the iOS app. This is already possible in the web interface, I don't know about the android version..

Platform:

- [ ]  Server
- [ ]  Web
- [x]  Mobile

**Comment by DrD4ffy** [comment]
It's missing for albums you create and person albums.

**Comment by bo0tzz** [comment]
#3789
